packages/sessions-sdk/src/index.ts

```diff
@@ -248,7 +272,8 @@ type EstablishSessionResult = ReturnType<
 >;
 
 export type Session = {
-  sessionKey: Keypair;
+  sessionKeyAddress: Address;
```

Comment:
seems inconsistent to store this as `Address` instead of `PublicKey`

Reply:
Ah good call.

Frankly this is a bit clunky in general -- we expose the publickey in two separate formats.  I am considering just exporting a function that takes the session and returns the legacy `PublicKey` rather than making it a field